packages/api-kit/tests/e2e/addSafeOperation.test.ts

```diff
@@ -1,15 +1,19 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
-import { ethers } from 'ethers'
 import sinon from 'sinon'
 import sinonChai from 'sinon-chai'
 import Safe from '@safe-global/protocol-kit'
 import SafeApiKit from '@safe-global/api-kit/index'
+import { getAddSafeOperationProps } from '@safe-global/api-kit/utils/safeOperation'
 import { Safe4337Pack } from '@safe-global/relay-kit'
+import * as viem from '@safe-global/relay-kit/node_modules/viem'
```

Comment:
This import may be wrong

Reply:
I know the import looked weird but it was actually intentionally, because it fixed an error that I was getting with a regular `import viem from 'viem'` statement:
```
Error: Trying to stub property 'createPublicClient' of undefined
```

➡️ https://github.com/safe-global/safe-core-sdk/actions/runs/9856259113/job/27212869062?pr=887